e2e-polybft/e2e/skyline_bridge_mba_test.go

```diff
@@ -263,7 +263,11 @@ func TestE2E_SkylineBridgeMBA_UTxOConsolidation(t *testing.T) {
 			map[string][]string{
 				cardanofw.ChainIDCardano: {cardanofw.ChainIDPrime},
 				cardanofw.ChainIDPrime:   {cardanofw.ChainIDCardano},
-			}, sendtx.BridgingTypeCurrencyOnSource,
+			},
+			map[string]sendtx.BridgingType{
```

Comment:
What if there are two bridgings from the Prime chain to Nexus and Vector? Why would we want both of them to be of the same bridging type? If we already allow multiple types per test, why not make it fully configurable?